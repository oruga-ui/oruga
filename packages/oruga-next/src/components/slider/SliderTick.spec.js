import { mount } from '@vue/test-utils'
import OSlider from '@components/slider/Slider'
import OSliderTick from '@components/slider/SliderTick'

let wrapper
let wrapperParent

const WrapperComp = {
    template: `
        <OSlider>
            <OSliderTick ref="tick" />
        </OSlider>`,
    components: {
        OSlider, OSliderTick
    }
}

describe('OSliderTick', () => {
    beforeEach(() => {
        wrapperParent = mount(WrapperComp)
        wrapper = wrapperParent.find({ ref: 'tick' })
    })

    it('is called', () => {
        expect(wrapper.exists()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
