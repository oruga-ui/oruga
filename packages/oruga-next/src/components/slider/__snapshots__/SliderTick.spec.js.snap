// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OSliderTick render correctly 1`] = `
<div class="o-slide__tick o-slide__tick--hidden" style="left: 0%;">
    <!---->
</div>
`;
